### PR TITLE
Dev flynn

### PR DESCRIPTION
**Pull request by UlrichFlynn**
- Implements ProTips card

### CODE REVIEW

File: src/components/index.ts [hunk 1]
```diff
@@ -2,5 +2,6 @@ import { ListItem } from "./buttons";
 import ToggleLightDarkMode from "./buttons/toggle-light-dark-mode";
 import VendorNavbar from "./vendor-navbar";
 import DashboardSideBar from "./vendor-sidebar";
+import ProTips from "./pro-tips";
```
Comment (ideal-lovette) [inline src/components/index.ts]: Could you name this `pro-tips.tsx` as `ProTips.tsx` as it is a JSX file?

File: src/components/pro-tips/index.tsx [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    );
+}
+
```
Comment (ideal-lovette) [inline src/components/pro-tips/index.tsx]: Throughout your component I've not seen any place where you used the `dark` keyword in your `className` how have you tackled dark/light mode behaviours?

Comment (ideal-lovette) [reply src/components/pro-tips/index.tsx]: If you have any difficulties implementing dark mode light mode let me know

File: src/components/index.ts (hunk 1 above)
Comment (UlrichFlynn) [reply src/components/index.ts]: You mean I should rename it as /pro-tips/ProTips.tsx?

File: src/components/pro-tips/index.tsx (hunk 2 above)
Comment (UlrichFlynn) [reply src/components/pro-tips/index.tsx]: Please I have difficulties implementing light/dark mode.

**Review by AkonehSilas — DISMISSED** [review]

**Comment by ideal-lovette** [comment]
Hello @UlrichFlynn  please resolve these merge conflicts and follow up to make sure that it is merged

**Comment by ideal-lovette** [comment]
Get to me if you have any challenges

**Review by ideal-lovette — DISMISSED** [review]

File: src/components/ProTips/index.tsx [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    <div className="grid grid-cols-1 md:grid-cols-2 gap-12 mb-8">
+                        <div className="flex">
+                            <div className="outline outline-gray-200 rounded-full w-16 h-16 me-3">
+                                <div className="m-5">
+                                    <BsTelephone size='1.5rem' color={`${isDarkMode ? 'white' : 'black'}`} />
```
Comment (ideal-lovette) [inline src/components/ProTips/index.tsx]: Hey @UlrichFlynn, what you've done here is correct. But you could change icon font-color in darkmode/ lightmode in it's className attribute like below

`<BsTelephone size='1.5rem' className="text-gray-800 dark:text-white" />`

File: src/components/ProTips/index.tsx [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+                                    <div className="flex outline outline-gray-200 outline-1 py-1 rounded">
+                                        <img className="object-cover w-5 h-5 rounded mr-1" src="/logo192.png" alt="" />
+                                        <p className='text-sm font-bold text-gray-400'>Time</p>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                        <div className="flex">
+                            <div className="outline outline-gray-200 rounded-full w-16 h-16 me-3">
+                                <div className="m-5">
```
Comment (ideal-lovette) [inline src/components/ProTips/index.tsx:111]: Same thing on this line

**Review by ideal-lovette — CHANGES_REQUESTED** [review]

File: src/components/ProTips/index.tsx (hunk 3 above)
Comment (UlrichFlynn) [reply src/components/ProTips/index.tsx]: Ok

**Review by ideal-lovette — APPROVED** [review]

**Pull request merged**